targets/csharp/templates/PlayFabSettings.cs.ejs

```diff
@@ -34,12 +40,36 @@ namespace PlayFab
         public static readonly string AD_TYPE_IDFA = "Idfa";
         public static readonly string AD_TYPE_ANDROID_ID = "Adid";
 <% } %>
-        public static string GetFullUrl(string apiCall)
+        public static string GetFullUrl(string apiCall, Dictionary<string, string> getParams)
         {
+            StringBuilder sb = new StringBuilder(1000);
+        
             var baseUrl = UseDevelopmentEnvironment ? DevelopmentEnvironmentUrl : ProductionEnvironmentUrl;
-            if (baseUrl.StartsWith("http"))
-                return baseUrl;
-            return "https://" + TitleId + baseUrl + apiCall;
+            if (!baseUrl.StartsWith("http"))
+                sb.Append("https://").Append(TitleId);
+            sb.Append(baseUrl);
+            if (!baseUrl.StartsWith("http"))
```

Comment:
I think this second if statement is incorrect.
You always need the apiCall added to the string.
Remove line 51, and fix the tabbing for (current) line 52.